ni_measurementlink_service/measurement/info.py

```diff
@@ -44,11 +43,16 @@ class ServiceInfo(NamedTuple):
         For e.g., ni.measurementlink.measurement.v2.MeasurementService.
         Defaults to ["ni.measurementlink.measurement.v1.MeasurementService"].
 
+        annotations (Dict<str,str>): Dict that contains extra information of the measurement.
```

Comment:
Python doesn't use C++/C# syntax.

```suggestion
        annotations (Dict[str,str]): Dict that contains extra information of the measurement.
```